src/apps/organisations/components/OrganisationForm.tsx

```diff
@@ -0,0 +1,188 @@
+import React, { useRef } from "react";
+import { Link } from 'react-router-dom';
+import {
+  Button,
+  FormControl,
+  InputLabel,
+  makeStyles,
+  MenuItem,
+  Typography
+} from "@material-ui/core";
+import { Organisation, Extras } from '../types';
+import { ErrorMessage, Field, Form, Formik } from "formik";
+import * as Yup from "yup";
+import { Select, TextField } from "formik-material-ui";
+
+
+interface Props {
+  onSubmit?: Function;
+  loading?: boolean;
+  status?: string;
+  errors?: {};
+  context?: string;
+}
+const ExtrasSchema = Yup.object().shape<Extras>({
```

Comment:
I think we can dispense with having a whole type and schema defined for `extras` when we're really just treating it as a pass-through string. I actually would be inclined to keep this out of the UI altogether until we have a definitive reason for reading or writing this metadata. The `ExtrasSchema` we use in the `concept` app is actually used to store a range of metadata important to OpenMRS, but which doesn't fit into the default OCL API (mostly because it only applies to numerical concepts). Here, even if OCL supports additional metadata, I don't think we actually use it for anything (and hand-editing a JSON document in a field is not a great idea).

Reply:
Sounds good